src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -129,7 +129,13 @@ case class CsvRelation protected[spark] (
               s"Dropping malformed line: ${tokens.mkString(",")}")
             None
           case pe: java.text.ParseException if dropMalformed =>
```

Comment:
Also please add this new exception handling case to `buildScan(requiredColumns: Array[String])` as well.